src/components/Header/Search.jsx

```diff
@@ -132,10 +132,10 @@ const Search = () => {
   }, [handleKeyDown]);
 
   const fetchTerms = useCallback(debounce(async (searchTerm) => {
-    const data = await searchAll(searchTerm);
-    const dataTerms = data?.results.filter(result => result.type === "TERM")
-    const dataOrganizations = data?.results.filter(result => result.type === "ORGANIZATION")
-    const dataOntologies = data?.results.filter(result => result.type === "ONTOLOGY")
+    const data = await elasticSearch(searchTerm);
+    const dataTerms = data?.results.filter(result => result.type === "term")
+    const dataOrganizations = data?.results.filter(result => result.type === "organization")
```

Comment:
factor out the types 'term', 'organization' and all the others in an external enum.